modules/pro-install/assets/js/pro-install-events.js

```diff
@@ -0,0 +1,100 @@
+window.trackUpgradeNowClickEvent = function() {
+	if ( ! window.elementorCommon?.config?.experimentalFeatures?.editor_events ) {
+		return;
+	}
+
+	const eventsManager = window.elementorCommon?.eventsManager || {};
+	const dispatchEvent = eventsManager.dispatchEvent?.bind( eventsManager );
+
+	const eventName = 'upgrade_now_click';
+	const eventData = {
+		app_type: 'editor',
+		location: 'Elementor WP-admin pages',
+		secondaryLocation: 'Connect account page',
+		trigger: 'click',
+	};
+
+	dispatchEvent?.( eventName, eventData );
+};
+
+window.trackConnectAccountEvent = function() {
+	if ( ! window.elementorCommon?.config?.experimentalFeatures?.editor_events ) {
+		return;
+	}
+
+	const eventsManager = window.elementorCommon?.eventsManager || {};
+	const dispatchEvent = eventsManager.dispatchEvent?.bind( eventsManager );
+
+	const eventName = 'connect_account';
+	const eventData = {
+		app_type: 'editor',
+		location: 'Elementor WP-admin pages',
+		secondaryLocation: 'Connect account page',
+		trigger: 'click',
+	};
+
+	dispatchEvent?.( eventName, eventData );
+};
+
+window.trackOpenConnectPageEvent = function() {
+	if ( ! window.elementorCommon?.config?.experimentalFeatures?.editor_events ) {
+		return;
+	}
+
+	const eventsManager = window.elementorCommon?.eventsManager || {};
+	const dispatchEvent = eventsManager.dispatchEvent?.bind( eventsManager );
+
+	const eventName = 'open_connect_page';
+	const eventData = {
+		app_type: 'editor',
+		location: 'Elementor WP-admin pages',
+		secondaryLocation: 'Connect account page',
+		trigger: 'page_load',
+	};
+
+	dispatchEvent?.( eventName, eventData );
+};
+
+window.trackProInstallEvent = function() {
+	if ( ! window.elementorCommon?.config?.experimentalFeatures?.editor_events ) {
+		return;
+	}
+
+	const eventsManager = window.elementorCommon?.eventsManager || {};
+	const dispatchEvent = eventsManager.dispatchEvent?.bind( eventsManager );
+
+	const eventName = 'pro_install';
+	const eventData = {
+		app_type: 'editor',
+		location: 'Elementor WP-admin pages',
+		secondaryLocation: 'Connect account page',
+		trigger: 'click',
+	};
+
+	dispatchEvent?.( eventName, eventData );
+};
+
+document.addEventListener( 'DOMContentLoaded', () => {
+	window.trackOpenConnectPageEvent();
+
+	const upgradeButton = document.querySelector( '.button-upgrade' );
+	if ( upgradeButton ) {
+		upgradeButton.addEventListener( 'click', () => {
```

Comment:
Can't we just pass the function directly here, instead of creating a new anonymous function around it?
There are no other calls or logic inside these functions anyway.